scripts/bug_classifier.py

```diff
@@ -49,8 +49,10 @@ def classify_bugs(model_name: str, classifier: str, bug_id: int) -> None:
         bugs = bugzilla.get_bugs()
 
     for bug in bugs:
-        print(
-            f'https://bugzilla.mozilla.org/show_bug.cgi?id={bug["id"]} - {bug["summary"]} '
+        logger.info(
+            "https://bugzilla.mozilla.org/show_bug.cgi?id=%s - %s ",
+            bug["id"],
+            bug["summary"],
```

Comment:
This part is an interactive program; it is more suitable to use `print()`. Please drop the changes in this file.